### ISSUE TITLE
Service blocks sleep on Windows 11

### ISSUE THREAD

**Issue by djh816** [devcompl/TTController#215]
**Describe the issue**
Something seems to have changed either in latest betas or with Windows 11 but this service now unfortunately keeps my machine awake no matter what :( I have tried reinstalling and also tried changing my Razor chrome linked/advertised PSU to "one time" trigger rather than always trigger but no difference. Currently I have worked around this by using TaskScheduler to kill the service and then set it back to autostart so on reboot it starts successfully again.

**Debug report**
No debug report, but logs look normal during startup/shutdown of service and nothing indicating why host wont sleep. This applies to both "screensaver" display sleep as well as "power" host sleep. Something about this service seems to keep Windows 11 awake.

**Current configuration**
```json
{
  "Profiles": [
    {
      "Name": "PSU",
      "Ports": [
        [9802, 9001]
      ],
      "Effects": [
        {
          "Type": "RazerConnectEffect",
          "Config": {
            "Trigger": {
              "Type": "AlwaysTrigger"
            }
          }
        }
      ]
    }
  ],
  "PortConfigs": [
    {
      "Ports": [
        [9802, 9001],
      ],
      "Config": {
        "ColorModifiers": [
          {
            "Type": "LerpLedColorModifier"
          }
        ]
      }
    }
  ]
}
```

**Comment by djh816** [comment]
For anyone curious I have the following scheduled tasks (probably could be cleaner..). It's four actions total, two for startup and two for login just in case. A bit hacky but seems to work reliably for now.

At system startup (run whether user is logged on or not, run with highest privs, hidden, Windows 10):
1) Trigger: Startup, Actions:
Start a program - sc config "TTController" start=auto
Start a program - sc start "TTController"

2) Trigger: Startup with 1 minute delay, Actions:
Start a program - sc stop "TTController"
Start a program - sc config "TTController" start=auto

At log on of main user (run whether user is logged on or not, run with highest privs, hidden, Windows 10):

3) Trigger: At logon of user, Actions:
Start a program - sc config "TTController" start=auto
Start a program - sc start "TTController"

4) Trigger: At logon of user with 1 minute delay, Actions:
Start a program - sc stop "TTController"
Start a program - sc config "TTController" start=auto

**Comment by djh816** [comment]
I love this service and its shit that my gpu wont let me disable all RGB but not sleeping is unacceptable in my usecase (OLED, and this single app holds a static image...)

Thanks for a decent service but I'm out. Hopefully someone can pick this up. TBH given my display I'm disabling whatever entirely, leaving my shit PSU in rgb cycle. I'll buy a new one etc, my MSI MPG gen OLED isn't worth the hassle.

TLDR: In current state this forces ALL displays to be on (to my knowledge) per Windows sleep nuances
